packages/tools/devtools/devtools-view/src/test/SettingsView.test.tsx

```diff
@@ -0,0 +1,20 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+// eslint-disable-next-line import/no-unassigned-import
+import "@testing-library/jest-dom";
+import { render } from "@testing-library/react";
+import * as axe from "axe-core";
+import React from "react";
+
+import { SettingsView } from "../components/index.js";
+
+describe("SettingsView Accessibility Check", () => {
+	it("SettingsView is accessible", async () => {
+		const { container } = render(<SettingsView />);
+		const results = await axe.run(container);
+		expect(results.violations.length).toBe(0);
```

Comment:
Would be nice to print the violations if the assertion fails, so we can see what needs to be fixed.

Reply:
Since we're expecting lots of tests like this, we should probably create a utilities module to share some axe-related helpers. A `assertNoAccessabilityViolations` helper that did this assert and nicely printed error messages about the violations would be very useful.